.gitignore

```diff
@@ -1 +1,5 @@
 /build
+.cproject
```

Comment:
These should go in your `~/.gitignore` file instead of the project-specific one. You might have to edit ~/.gitconfig to include:

```ini
[core]
	excludesfile = ~/.gitignore
```